### PR TITLE
Check if the given address is well-formed

### LINKED ISSUE #11: Validate `address` argument
Check if the given address is well-formed.

### PR DESCRIPTION
**Pull request by redongjun**
Fix #11

### CODE REVIEW

**Comment by kseo** [comment]
@redongjun Would you resolve conflicts?

**Comment by redongjun** [comment]
@kseo I fixed it. 😄

File: src/index.ts [hunk 1]
```diff
@@ -234,13 +238,17 @@ function parseAccountType(accountType: string): AccountType {
...
         throw new CLIError(CLIErrorType.OptionRequired, {
             optionName: "address"
         });
     }
-    // FIXME: Validate the address.
+    if (accountType === "platform") {
+        PlatformAddress.fromString(address);
+    } else {
+        AssetTransferAddress.fromString(address);
+    }
```
Comment (majecty) [inline src/index.ts]: We should catch error from `PlatformAddress.fromString` and `AssetTransferAddress.fromString`, and rethrow CLIError.
Also, we should add a proper enum value to `CLIErrorType`. And print a proper error message by adding the message in `getErrorMessage` in the error.ts file.

Comment (redongjun) [reply src/index.ts]: @majecty Ok, I will fix it.

**Comment by redongjun** [comment]
@majecty How about printing out the list of addresses and selecting it? It seems better for `export` and `delete` commands.

**Comment by majecty** [comment]
@redongjun Good idea.
When you working on it, please consider this.
When the CLI is not running in the interactive shell, we should get an address from the args.
When only the CLI is running in the interactive shell, we can use an interactive feature.

Please create an issue about it.

**Pull request merged**